doc/swish/osi.tex

```diff
@@ -422,6 +430,26 @@
 \var{pid}. It returns \code{\#t} when successful and an error pair
 otherwise.
 
+\defineentry{osi\_signal\_start}
```

Comment:
Oops! Thanks for catching this.

Aside: it would be nice to fix our LaTeX setup so we don't have to escape underscores in these cases. It'll make grep easier.